### DISCUSSION TITLE
Do you plan to make some maps more enjoyable?

### DISCUSSION THREAD

**Discussion by Warentan** [space-wizards/space-station-14#18468] [Questions]
There are two common maps, Core and Barratry, that the community _completely despises_. We have dubbed Barratry as "Detroit."  Barratry doesn't even have a singularity, and last time I played on core I died within 5 minutes. Do you plan to make these or other maps a bit more bearable to live in?

**Comment by Vaaankas** [comment]
Thats the whole point of those maps

**Comment by Androclast** [comment]
1. When you map it

2. Both stations are unique designs that introduce variety to the game instead of just having 4 different flavors of box and be done with it.
3. Both maps hinge on the belief that engi knows what "construction" is. Barratry with its rundown aesthetic (literally in the name) and Core with the requirement of using the singularity and tcomms setup.
4. Barratry has a singularity, the parts are nearby and you have to build it. Aspid lacks the singularity.

**Comment by metalgearsloth** [comment]
Can you link me to where you left feedback for this map?
